back/app/services/xlsx_export/input_sheet_generator.rb

```diff
@@ -170,11 +170,10 @@ def meta_report_fields
           meta_fields << likes_count_report_field
           meta_fields << dislikes_count_report_field
         end
-        if participation_method.supports_baskets?
-          meta_fields << baskets_count_report_field
-          meta_fields << votes_count_report_field
-        end
-        meta_fields << budget_report_field if participation_method.supports_budget?
+        meta_fields << baskets_count_report_field('picks') if participation_method.additional_export_columns.include? 'picks'
```

Comment:
What's the difference between "picks" and "participants". It looks like both are included but have identical values?

Reply:
Yeah, just different labels for the same field which make more sense depending on the voting method - neither ever get included in the same sheet